Sm/stage2

sayancoding opened the pull request.

sounishnath003 reviewed (changes requested):
Well structured, but make sure, that the code never fails, cause the dialog to submit button isn't pop up on the screen. again. while making service handle those good kinds of stuff carefully.

sayancoding commented:
Now fixed. I've added all dialog's close and submit btn to programmatically
close up and also make the student teacher invite popup.

On Tue, Sep 1, 2020 at 11:17 AM Sounish Nath <notifications@github.com>
wrote:

> *@sounishnath003* requested changes on this pull request.
>
> Well structured, but make sure, that the code never fails, cause the
> dialog to submit button isn't pop up on the screen. again. while making
> service handle those good kinds of stuff carefully.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sounishnath003/Educatify/pull/11#pullrequestreview-479416455>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIS6UHA7LYLG7MH5LJIJ3TLSDSDFNANCNFSM4QPYQTRA>
> .
>
